# Default Exception Handler
a

try:
    a
except NameError:
    print("a is not defined")
finally:
    print("Runs no matter exception occur or not")

assert 1 == 1

assert 1 == 2

with open("../README.md") as f:
    pass

